### DISCUSSION TITLE
Add space or a comma after each <tr> and <td> element

### DISCUSSION THREAD

**Discussion by capdavid** [html-to-text/node-html-to-text#313] [Q&A] [answered]
Hello, I am using this beautiful parsing library to get all text from html. But I have a problem with tables, because between rows and columns, there is nothing so the text looks like this: Cell1Cell2Cell3. I want to have `Cell 1, Cell 2, Cell 3`

How can I achieve this?

**Comment by KillyMXI** [comment, marked as answer]
First of all, it seems like you're using it with the default format for tables.

In case you need CSV-like output, refer to https://github.com/html-to-text/node-html-to-text/issues/242#issuecomment-1357882932 and change tab to the separator you need.
There is no built-in formatter like that currently, so there is just a bit more configuration with a custom formatter.

In case you just need tables to be formatted visually:
```js
{
  selectors: [
    { selector: 'table', format: 'dataTable' }
  ]
}
```

**Reply by capdavid** [reply]
Yes this works well for me, thank you!
